.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+#### Checklist template for LEADERS
+
+&#x2611; | &#x2610;
+
+&#x2610; New code is covered with tests and coverage requirements are met.
+&#x2610; Build is not failing.
+&#x2610; All tests are passing.
+&#x2610; All conflicts are resolved and the branch is rebased on top of the up-to-date parent branch.
+&#x2610; All issue requirements are implemented.
+&#x2610; At least two reviewers are assigned.
+&#x2610; The Pull request is linked to the related issue.
+&#x2610; Correct branch name. Convention [https://github.com/bycza-zagroda/track-expenses-app-documentation/blob/develop/githubwork/GITHUB_WORK.md]
+&#x2610; Correct unit test name - suffix 'Test' in the end.
+&#x2610; Correct integration test name - suffix 'IT' in the end.
+&#x2610; Postivie and negative path for tests.
```

Comment:
Maybe "Positive and negative test paths have been implemented"?

Reply:
Done